packages/xchain-util/src/asset.ts

```diff
@@ -165,48 +164,42 @@ export const formatAssetAmount = ({
 export const formatBaseAmount = (amount: BaseAmount) => formatBN(amount.amount(), 0)
 
 /**
- * Base "chain" asset of Avalanche chain.
- *
- */
-export const AssetAVAX: Asset = { chain: Chain.Avalanche, symbol: 'AVAX', ticker: 'AVAX', synth: false }
-
-/**
- * Base "chain" asset of Binance chain.
+ * Base "chain" asset on bitcoin cash main net.
  *
  * Based on definition in Thorchain `common`
  * @see https://gitlab.com/thorchain/thornode/-/blob/master/common/asset.go#L12-24
  */
-export const AssetBNB: Asset = { chain: Chain.Binance, symbol: 'BNB', ticker: 'BNB', synth: false }
+export const AssetBCH: Asset = { chain: 'BCH', symbol: 'BCH', ticker: 'BCH', synth: false }
```

Comment:
Remove all `const Asset{XYZ}` from `asset.ts`. If you do need these in tests, define it there, but don`t export. _I've read your message about at Discord..._